lib/xain.ex

```diff
@@ -200,4 +200,22 @@ defmodule Xain do
   defp set_defaults(attrs, name) do
     Keyword.merge(get_defaults(name), attrs)
   end
+
+  defp html_escape(html) do
```

Comment:
I think we should add plug as a dependency and use Plug.HTML.escape_html instead of creating our own here. @romul what do you think?